go.mod

```diff
@@ -896,7 +896,9 @@ replace (
 	// https://github.com/grafana/cadvisor/tree/grafana-v0.47-noglobals
 	github.com/google/cadvisor => github.com/grafana/cadvisor v0.0.0-20240729082359-1f04a91701e2
 
-	github.com/prometheus-community/postgres_exporter => github.com/grafana/postgres_exporter v0.15.1-0.20240417113938-9358270470dd
+	// TODO(dehaansa): integrate the changes from the exporter-package-v0.15.0 branch into at least the
+	// grafana fork of the exporter, or completely into upstream
+	github.com/prometheus-community/postgres_exporter => github.com/grafana/postgres_exporter v0.15.1-0.20241105053755-e0a51174f168
```

Comment:
your branch was [merged to master](https://github.com/grafana/postgres_exporter/pull/23) so the first part of the todo should be done already, right? 
do you plan on creating the PR upstream? Even if we have our own fork with our own changes, if the bugfix also applies upstream it would be nice to contribute

Reply:
My branch was merged to master, but I discovered that the version we were using previously is from an un-merged branch from Piotr & another developer at grafana: https://github.com/grafana/postgres_exporter/compare/master...grafana:postgres_exporter:exporter-package-v0.15.0?expand=1 . I merged master into that branch to get my changes into the branch that's compatible with alloy right now, and was planning to jump back to cleaning up their branch and getting it merged later.